reference/configuration/tikv-server/configuration-file.md

```diff
@@ -94,7 +94,7 @@ This document only describes the parameters that are not included in command-lin
 
 ## readpool.unified
 
-Configuration items related to the single thread pool serving read requests.
+Configuration items related to the single thread pool serving read requests. This thread pool supersedes the original storage thread pool and coprocessor thread pool in the 4.0 version.
```

Comment:
```suggestion
Configuration items related to the single thread pool serving read requests. This thread pool supersedes the original storage thread pool and coprocessor thread pool since the 4.0 version.
```